.gitignore

```diff
@@ -287,4 +287,7 @@ __pycache__/
 *.odx.cs
 *.xsd.cs
 
-.publish/
\ No newline at end of file
+.publish/
+
+# VSCode
+.vscode
```

Comment:
This should actually be checked in

Reply:
I didn't actually use VSCode to debug. So not sure my configuration is valid. Let me simply revert it.